src/utils/jsx-entity.ts

```diff
@@ -338,7 +340,8 @@ export const commonInflators: Required<{ [K in keyof ComponentData]: InflatorFn
   grabbable: inflateGrabbable,
 
   // inflators that create Object3Ds
-  directionalLight: inflateDirectionalLight
+  directionalLight: inflateDirectionalLight,
+  billboard: createDefaultInflator(Billboard)
```

Comment:
nit: There's a comment that separates inflators that do / don't create `Object3D`s. Not sure this comment is useful, but we should either remove the comment or make sure it's accurate (by moving `billboard` above it)

Reply:
I'll leave the comment for now and move the creator above.